### PR TITLE
Methods to query the metrics (as in DSS Metrics) of a MES or a ME

### PR DESCRIPTION
**Pull request by lpenet**
Companion of https://github.com/dataiku/dip/pull/12419

### CODE REVIEW

**Review by FChataigner — APPROVED** [review]
LGTM

File: dataikuapi/dss/modelevaluationstore.py [hunk 1]
```diff
@@ -171,6 +174,53 @@ def create_model_evaluation(self, labels=None, prediction_type=None, model_type=
...
+        Returns:
+            a list of metric objects and their value
+        """
+        return ComputedMetrics(self.client._perform_json(
+            "GET", "/projects/%s/modelevaluationstores/%s/metrics/last" % (self.project_key, self.mes_id)))
+
+
+    def get_metric_history(self, metric):
+        """
+        Get the history of the values of the metric on this dataset
```
Comment (fterrazzoni) [inline dataikuapi/dss/modelevaluationstore.py:194]: dataset => model evaluation

File: dataikuapi/dss/modelevaluationstore.py [hunk 2]
```diff
@@ -171,6 +174,53 @@ def create_model_evaluation(self, labels=None, prediction_type=None, model_type=
...
+            an object containing the values of the metric, cast to the appropriate type (double, boolean,...)
+        """
+        return self.client._perform_json(
+            "GET", "/projects/%s/modelevaluationstores/%s/metrics/history" % (self.project_key, self.mes_id),
+            params={'metricLookup': metric if isinstance(metric, str)or isinstance(metric, unicode)
+                                           else json.dumps(metric)})
+
+    def compute_metrics(self, metric_ids=None, probes=None):
+        """
+        Compute metrics on this managed folder. If the metrics are not specified, the metrics
```
Comment (fterrazzoni) [inline dataikuapi/dss/modelevaluationstore.py:206]: managed folder => model evaluation

**Review by fterrazzoni — COMMENTED** [review]
Just wrong comments

File: dataikuapi/dss/modelevaluationstore.py (hunk 1 above)
Comment (lpenet) [reply dataikuapi/dss/modelevaluationstore.py:194]: ok.

**Pull request merged**